src/components/dialogs/parameters/non-evacuated-energy/non-evacuated-energy-parameters-selector.js

```diff
@@ -0,0 +1,200 @@
+        { label: 'GeneratorsCappings' },
+        { label: 'MonitoredBranches' },
+        { label: 'Contingencies' },
+    ];
+
+    const [rowDataGenerationStages, useFieldArrayOutputGenerationStages] =
+        useCreateRowDataSensi(
+            nonEvacuatedEnergyParam.NonEvacuatedEnergyGenerationStages
+        );
+
+    const [
+        rowDataGenerationStagesSelection,
+        useFieldArrayOutputGenerationStagesSelection,
+    ] = useCreateRowDataSensi(
+        nonEvacuatedEnergyParam.NonEvacuatedEnergyStagesSelection
+    );
+
+    const [rowDataGeneratorsCappings, useFieldArrayOutputGeneratorsCappings] =
+        useCreateRowDataSensi(
+            nonEvacuatedEnergyParam.NonEvacuatedEnergyGeneratorsCappings
+        );
+
+    const [rowDataMonitoredBranches, useFieldArrayOutputMonitoredBranches] =
+        useCreateRowDataSensi(
+            nonEvacuatedEnergyParam.NonEvacuatedEnergyMonitoredBranches
+        );
+
+    const [rowDataContingencies, useFieldArrayOutputContingencies] =
+        useCreateRowDataSensi(
+            nonEvacuatedEnergyParam.NonEvacuatedEnergyContingencies
+        );
+
+    const getColumnsDefinition = useCallback(
+        (columns) => {
+            if (columns) {
+                return columns.map((column) => ({
+                    ...column,
+                    label: intl.formatMessage({ id: column.label }),
+                }));
+            }
+            return [];
+        },
+        [intl]
+    );
+
+    return (
+        <>
+            <Grid sx={{ width: '100%' }}>
+                <Tabs value={tabValue} onChange={handleTabChange}>
+                    {tabInfo.map((tab, index) => (
+                        <Tab
+                            key={tab.label}
+                            label={<FormattedMessage id={tab.label} />}
+                            value={index}
+                            sx={{
+                                fontSize: 17,
+                                fontWeight: 'bold',
+                                textTransform: 'capitalize',
+                            }}
+                        />
+                    ))}
+                </Tabs>
+
+                {tabInfo.map((tab, index) => (
+                    <TabPanel key={tab.label} value={tabValue} index={index}>
+                        {tabValue === TAB_VALUES.GenerationStages && (
+                            <SensitivityTable
+                                arrayFormName={`${NonEvacuatedEnergyGenerationStages.name}`}
+                                columnsDefinition={getColumnsDefinition(
+                                    nonEvacuatedEnergyParam.COLUMNS_DEFINITIONS_STAGES
+                                )}
+                                useFieldArrayOutput={
+                                    useFieldArrayOutputGenerationStages
+                                }
+                                createRows={rowDataGenerationStages}
+                                tableHeight={367}
```

Comment:
does this still hold with the new parameters design? it looks like there is a vertical scroll bar now, can we use flex box or similar to avoid hard coded height in the code?

Reply:
for example with "295" I don't have a verticl scroll bar anymore for the 100% zoom level

Reply:
ok. changed